package/MDAnalysis/core/topologyobjects.py

```diff
@@ -239,8 +238,7 @@ def angle(self):
         """
```

Comment:
Is the note in the docstring still applicable regarding the low precision? And if not, would it warrant making the corresponding tests stricter in that regard?